packages/admin/src/Navigation/NavigationItem.php

```diff
@@ -3,28 +3,31 @@
 namespace Filament\Navigation;
 
 use Closure;
+use Filament\Support\Concerns\EvaluatesClosures;
 
 class NavigationItem
 {
-    protected ?string $group = null;
+    use EvaluatesClosures;
```

Comment:
@danharrin, should it extend our `Component` class instead of manually using the trait?